src/applications/personalization/dashboard-2/components/apply-for-benefits/ApplyForBenefits.jsx

```diff
@@ -121,7 +124,7 @@ const ApplyForBenefits = ({
       <ApplicationsInProgress />
       <BenefitsOfInterest showChildren={hasLoadedAllData}>
         <>
-          {showHealthCare && (
+          {hideHealthCareBenefitInfo ? null : (
```

Comment:
...and using the new booleans in JSX is nicer with the ternary. I think this now reads better than:

`{!hideHealthCareBenefitInfo && <BenefitOfInterest>...}`